crates/sui-types/src/object.rs

```diff
@@ -449,6 +449,15 @@ impl Object {
         (self.id(), self.version(), self.digest())
     }
 
+    pub fn compute_object_type(&self) -> ObjectType{
```

Comment:
compute imply getting the object type need computation and could be expansive (e.g. compute digest), we can simply call this `get_object_type`